### PR TITLE
add eupl license

### PR DESCRIPTION
**Pull request by koopa1338**
Added EUPL license to global snippets. The long lines can also be separated, but I wanted to keep it as original as I could.

### CODE REVIEW

**Comment by kerkmann** [comment]
lgtm; :)

**Review by kerkmann — APPROVED** [review]

**Review by OkelleyDevelopment — CHANGES_REQUESTED** [review]
Hey @koopa1338 :wave: 

Before `lgtm` can happen, could you double check the link (on line 389) in the PR? 

When I tested, it brought me to a 404 page which I doubt is the intent here :sweat_smile: 


Once that's resolved then we can get this merged! :sunglasses:

File: snippets/global.json [hunk 1]
```diff
@@ -374,5 +374,23 @@
...
+    "description": "EUPL License",
+    "body": [
+      "The EUPL License (EUPL)",
+      "",
+      "Copyright (c) ${CURRENT_YEAR} ${0:Author}",
+      "Licensed under the EUPL, Version 1.2 or – as soon they will be approved by the European Commission - subsequent versions of the EUPL (the ''Licence'');",
+      "You may not use this work except in compliance with the Licence.",
+      "You may obtain a copy of the Licence at:",
+      "",
+      "<https://joinup.ec.europa.eu/software/page/eupl5>",
```
Comment (kerkmann) [inline snippets/global.json:389]:
```suggestion
      "<https://joinup.ec.europa.eu/software/page/eupl>",
```
@OkelleyDevelopment Indeed, you're right. Seems like that the `5` was accidentally added. :)

**Review by kerkmann — CHANGES_REQUESTED** [review]

**Comment by koopa1338** [comment]
Fixed URL with a284e0a

**Review by OkelleyDevelopment — APPROVED** [review]
Epic thanks for checking @koopa1338  @kerkmann 

Lgtm! :sunglasses:

**Pull request merged**